.github/workflows/test-erlang-otp-21.3.yaml

```diff
@@ -0,0 +1,547 @@
+        run: |
+          branch_or_tag_name=${GITHUB_REF#refs/*/}
+          ! test -d ebin || touch ebin/*
+          make ct-rabbit_mgmt_test_unit base_rmq_ref=master current_rmq_ref=$branch_or_tag_name FULL=
+      - name: ON FAILURE ARCHIVE TESTS LOGS
+        if: failure()
+        run: |
+          make ct-logs-archive
+      - name: ON FAILURE UPLOAD TESTS LOGS ARTIFACT
+        # https://github.com/marketplace/actions/upload-artifact
+        uses: actions/upload-artifact@v2-preview
+        if: failure()
+        with:
+          name: ct-rabbit_mgmt_test_unit-logs
+          path: "*-ct-logs-*.tar.xz"
+  ct-stats:
+    needs: [checks]
+    # https://help.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#contexts
+    name: ct-stats
+    runs-on: ubuntu-18.04
+    steps:
+      - name: CHECKOUT REPOSITORY
+        uses: actions/checkout@v2
+      # https://github.com/marketplace/actions/setup-elixir
+      - name: CONFIGURE OTP & ELIXIR
+        uses: actions/setup-elixir@v1
+        with:
+          otp-version: 21.3
+          # https://github.com/elixir-lang/elixir/releases
+          elixir-version: 1.8.0
+      - name: CACHE DEPS
+        uses: actions/cache@v1
+        with:
+          path: deps
+          key: otp-21.3_git-${{ github.sha }}_deps
+      - name: RUN TESTS
+        run: |
+          branch_or_tag_name=${GITHUB_REF#refs/*/}
+          ! test -d ebin || touch ebin/*
+          make ct-stats base_rmq_ref=master current_rmq_ref=$branch_or_tag_name FULL=
+      - name: ON FAILURE ARCHIVE TESTS LOGS
+        if: failure()
+        run: |
+          make ct-logs-archive
+      - name: ON FAILURE UPLOAD TESTS LOGS ARTIFACT
+        # https://github.com/marketplace/actions/upload-artifact
+        uses: actions/upload-artifact@v2-preview
+        if: failure()
+        with:
+          name: ct-stats-logs
+          path: "*-ct-logs-*.tar.xz"
+  capture-tested-deps-versions:
+    needs:
+      - eunit
+      - ct-cache
+      - ct-clustering
+      - ct-clustering_prop
+      - ct-config_schema
+      - ct-listener_config
+      - ct-rabbit_mgmt_http
+      - ct-rabbit_mgmt_only_http
+      - ct-rabbit_mgmt_rabbitmqadmin
+      - ct-rabbit_mgmt_stats
+      - ct-rabbit_mgmt_test_db
+      - ct-rabbit_mgmt_test_unit
+      - ct-stats
+    runs-on: ubuntu-18.04
+    steps:
+      - name: CHECKOUT REPOSITORY
+        uses: actions/checkout@v2
+      - name: CACHE DEPS
+        uses: actions/cache@v1
+        with:
+          path: deps
+          key: otp-21.3_git-${{ github.sha }}_deps
+      - name: FORMAT GIT REF
+        # https://github.community/t5/GitHub-Actions/How-can-I-set-an-expression-as-an-environment-variable-at/m-p/41804/highlight/true#M4751
+        id: ref
+        run: |
+          branch_or_tag_name=${GITHUB_REF#refs/*/}
+          echo "::set-output name=branch_or_tag_name::$branch_or_tag_name"
+      - name: UPLOAD TO S3
+        if: github.ref == 'refs/heads/master'
```

Comment:
Let's comment this (for both workflows) and double-check that it works before we uncomment and merge.